frontend/src/scenes/insights/ActionSelectInfo.tsx

```diff
@@ -0,0 +1,57 @@
+import { Divider } from 'antd'
+import React from 'react'
+import { ActionType } from '~/types'
+
+export function ActionSelectInfo({ entity }: { entity: ActionType }): JSX.Element {
+    if (!entity) return null
+    return (
+        <div className="select-box-info">
+            {entity.steps &&
+                entity.steps.map((step, index) => (
+                    <div key={step.id}>
+                        <strong>
+                            Match group {index + 1}:{' '}
+                            {step.event && step.event[0] == '$'
+                                ? step.event[1].toUpperCase() + step.event.slice(2)
```

Comment:
this feels like worth extracting to a function + giving a name within the file.

E.g.

```
function canonicalEventName(event: string): string
```